src/fromager/commands/build.py

```diff
@@ -66,12 +73,19 @@ def build_sequence(
     the build order file.
 
     """
+    server.start_wheel_server(wkctx)
+
     with open(build_order_file, "r") as f:
         for entry in progress.progress(json.load(f)):
             dist_name = entry["dist"]
             dist_version = Version(entry["version"])
+
+            if skip_existing and _is_wheel_built(wkctx, dist_name, dist_version):
```

Comment:
It would be good to have an INFO log message here saying that the build is being skipped because the wheel exists.